packages/runtime/src/next/runtime-provider.tsx

```diff
@@ -0,0 +1,16 @@
+import { ComponentProps } from 'react'
+
+import { RuntimeProvider } from '../runtimes/react/components/RuntimeProvider'
+
+import { FrameworkProvider } from './components/framework-provider'
+
+export function ReactRuntimeProvider({
+  children,
+  ...props
+}: ComponentProps<typeof RuntimeProvider>) {
```

Comment:
Next.js-specific runtime provider.

Reply:
All good points, but this is a part of our [public API](https://docs.makeswift.com/developer/app-router/installation#create-makeswift-provider-component), and I'm deliberately staying API compatible in this iteration.